test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eu
+
+for folder in src/*/*
+do
+  src=$folder/src.rego
+  test=$folder/src_test.rego
+  # TODO: enforce coverage
+  # needs https://github.com/open-policy-agent/opa/issues/2562 to see what was not covered
+  # needs https://github.com/open-policy-agent/opa/issues/2139 to see verbose output when using coverage
+  echo "opa test $src $test"
+  opa test $src $test
```

Comment:
I would suggest using the opa docker image at the version currently used in Gatekeeper, rather than the locally installed binary which may not be current.
e.g.
```
  IMAGE_OPA=openpolicyagent/opa
  IMAGE_OPA_VERSION=0.24.0
```
...
```
  docker run -it -v "$(pwd)/${folder}":/work "${IMAGE_OPA}":"${IMAGE_OPA_VERSION}" test -v /work/src.rego /work/src_test.rego
```

Reply:
Actually just saw the github workflow downloads it so that's fine. But it looks like 0.24 is the current version, could you change it to that?

Reply:
updated

Reply:
I still see 0.21.0 in actions.yaml instead of 0.24.0.
Also @sozercan had pushed[ a commit to this branch with formatting fixes ](https://github.com/grosser/gatekeeper-library/commit/c52df57652043fdcce01a3665c1d16960e3ba770)that appears to have been lost. Do you mind recreating that along with the version update?
Everything LGTM otherwise.

Reply:
updated and brought back the formatting changes